src/components/ColorPicker.tsx

```diff
@@ -144,8 +144,10 @@ const Picker = ({
               (event.currentTarget as HTMLButtonElement).focus();
               onChange(_color);
             }}
-            title={`${_color} — ${keyBindings[i].toUpperCase()}`}
-            aria-label={_color}
+            title={`${t(`colors.${_color.replace("#", "")}`)}${
+              _color !== "transparent" ? ` (${_color})` : ""
```

Comment:
lets use [isTransparent](https://github.com/excalidraw/excalidraw/blob/master/src/utils.ts#L333) util to cover all cases for transparency ?

Reply:
Done. (There were other places that compared to the raw string in the file, so using the helper consistently everywhere now.) https://github.com/excalidraw/excalidraw/pull/3871/commits/8588d0d7f4636bf209f7abaa03aa538d5cc6f00a